code/datums/gamemode/factions/bloodcult/bloodcult_spells.dm

```diff
@@ -175,3 +200,44 @@
 		to_chat(user, "<span class='notice'>You retrace your steps, carefully undoing the lines of the [removed_word] rune.</span>")
 	else
 		to_chat(user, "<span class='warning'>There aren't any rune words left to erase.</span>")
+
+
+//SPELL III
+/spell/cult/blood_dagger
+	name = "Blood Dagger"
+	desc = "Solidify some blood into a sharp weapon. Slash at your enemies to steal their blood. Use the dagger to re-absorb the stolen blood."
+	hud_state = "cult_blooddagger"
+
+	invocation_type = SpI_NONE
+	charge_type = Sp_RECHARGE
+	charge_max = 0
+	range = 0
+	spell_flags = null
+	insufficient_holder_msg = ""
+	still_recharging_msg = ""
+
+	cast_delay = 0
+
+/spell/cult/blood_dagger/choose_targets(var/mob/user = usr)
```

Comment:
This needs the `spell_flags` SELF_CAST (I think)

Reply:
eh, I might add a few lines of code here but that's much less expensive than the parent choose_targets()